stdeb.cfg

```diff
@@ -1,5 +1,6 @@
 [DEFAULT]
 No-Python2:
 Depends3: python3-setuptools, python3-yaml
-Conflicts3: python-vcstool
+Conflicts3: python-vcstool, python3-vcstool
+Suite3: jammy noble bookworm trixie
 X-Python3-Version: >= 3.5
```

Comment:
The lowest python version that you can find in these set of versions is probably jammy with 3.10.x . We can probably drop or update the version here.

Reply:
3.9.x in the case of Bullseye

Reply:
The minimum supported python version indicated in setup.py is still 3.5 to support RHEL targets as well. Keeping these in sync is easier than focusing the constraints based on platform IMO.